src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;
+
+    Circle() {
```

Comment:
dont forget about access modifier

Reply:
not fixed, default isn't good option here